server/checks/PullRequestTasks.js

```diff
@@ -0,0 +1,82 @@
+import Check, { getPayloadFn } from './Check'
+import { PULL_REQUEST } from '../model/GithubEvents'
+import { logger } from '../../common/debug'
+
+const CHECK_TYPE = 'pullrequesttasks'
+const CONTEXT = 'zappr/pr/tasks'
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+const debug = logger(CHECK_TYPE)
+const createStatePayload = getPayloadFn(CONTEXT)
+
+export function hasOpenTasks(pr) {
```

Comment:
When printing debug messages, I'm doing it inside the loop and to seperate those lines from other PR's I need to include repoName and prNo so that prevents me from just including pr body.